### PR TITLE
test: Refactor ThemeLiveReloadIT

### LINKED ISSUE #9746: Fix ThemeLiveReloadIT
ThemeLiveReloadIT test randomly fails in CI 
https://teamcity.vaadin.com/project.html?projectId=Flow&testNameId=8131873608327225402&tab=testDetails

### PR DESCRIPTION
**Pull request by mshabarov**
Refactors ThemeLiveReloadIT so as to apply a new font and background colour simultaneously in the first place, and then delete the styles and fonts in the end of test. I.e. create all you need -> delete all, instead of create -> delete -> create. 

Fixes #9746, #10283

### CODE REVIEW

**Comment by mshabarov** [comment]
The validation passed 10/10 -> unlock this PR and hope to the best it's now not flaky.

**Comment by mshabarov** [comment]
This test should be reworked:
- adding a new file is coupled with changing the existing file. So the webpack reload might be triggered by changing the existing file.
- adding a stylesheet file to `components` folder is not tested.

File: flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java [hunk 1]
```diff
@@ -65,47 +65,45 @@ public void cleanUp() {
...
-        final WebElement htmlElement = findElement(By.tagName("html"));
-        Assert.assertNotEquals(RED_COLOR,
-                htmlElement.getCssValue("background-color"));
+        Assert.assertFalse(
+                "Red background is not expected before applying the styles",
+                isCustomBackGroundColor());
+        Assert.assertTrue(
+                "Expected styles.css file to be auto-generated by flow",
+                stylesCSSFile.exists());
+        Assert.assertFalse("Expected no styles.css file exists before "
```
Comment (pleku) [inline flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java:84]:
```suggestion
        Assert.assertFalse("Expected no test-styles.css file exists before "
```

**Review by pleku — APPROVED** [review]

**Comment by pleku** [comment]
>This test should be reworked:
adding a new file is coupled with changing the existing file. So the webpack reload might be triggered by changing the existing file.
adding a stylesheet file to components folder is not tested

So these should be done as part of the individual bug tickets we have (or other doesn't yet have an issuu on) @mshabarov  ?

**Comment by mshabarov** [comment]
> So these should be done as part of the individual bug tickets we have (or other doesn't yet have an issuu on)

Yes, let's fix it within the bug tickets. The issue with no live-reload upon adding a file to `my-theme/components` has a discussion in support channel, but has no ticket yet. So I'll create it and check if adding a new file in the theme folder root still works.

**Comment by mshabarov** [comment]
Here is the ticket https://github.com/vaadin/flow/issues/10544. 
Adding a new file to the custom theme folder (no matter where, to the root or to the subfolder) is completely broken due to some changes between Flow `6.0.0.alpha4` and `6.0.0.alpha5`.
While this PR fixes the flakiness of the test, it should be reworked within fixing the above issue.

**Pull request merged**